test/ValidateID.cpp

```diff
@@ -406,7 +406,7 @@ class OpTypeArrayLengthTest
     : public spvtest::TextToBinaryTestBase<::testing::TestWithParam<int>> {
  protected:
   OpTypeArrayLengthTest()
-      : position_{0, 0, 0}, diagnostic_(spvDiagnosticCreate(&position_, "")) {}
+      : position_(), diagnostic_(spvDiagnosticCreate(&position_, "")) {}
```

Comment:
I was under the impression that this will NOT zero-initialize which is why I made the comment. iirc, it will call the default-constructor, which in this case does nothing.